### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by shawn159**

### CODE REVIEW

File: CoCoding/apps/codes/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CodesConfig(AppConfig):
+    name = 'codes'
```
Comment (MyungSeKyo) [inline CoCoding/apps/codes/apps.py:5]: dude! you wanna die? have you decided to ruin our project????right?

Comment (JungChaeMoon) [inline CoCoding/apps/codes/apps.py:5]: Oh my god.... hey dude, you should write app full python path............

Comment (shawn159) [reply CoCoding/apps/codes/apps.py]: Oh thank you. You are so sweet guy

Comment (shawn159) [reply CoCoding/apps/codes/apps.py]: huk...I'm sorry my bad

**Pull request merged**